chaoslib/litmus/docker_service_kill/docker-service-kill.j2

```diff
@@ -0,0 +1,42 @@
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: service-kill-{{ run_id }}
+  labels:
+    app: service-kill
+    name: service-kill-{{ run_id }}  
+{% if chaos_uid is defined and chaos_uid != '' %}
+    chaosUID: {{ chaos_uid }}
+{% endif %}        
+spec:
+  nodeName: {{ node_name }}
+  restartPolicy: Never
+  containers:
+  - name: service-kill
+    image: ubuntu:16.04
+    command: ["/bin/bash"]
+    args: ["-c", "sleep 10 && echo injecting_docker_crash.. && systemctl stop docker && sleep {{ c_duration }} && systemctl start docker && echo reverted_docker_crash.."]
```

Comment:
would you like to echo/print a conventional message(`echo "injecting docker crash..."`), or underscores are placed intentionally?

Reply:
ok conventional messaging seems to be right idea.